app/views/start_group/new.html.haml

```diff
@@ -0,0 +1,59 @@
+.row
+  .col-md-6.col-md-offset-3.col-sm-10.col-sm-offset-1
+    .row
+      %h1= t :"start_group_form.header"
+      = form_for @group, url: start_group_path, html: {class: 'form-horizontal'} do |f|
+        .form-group
+          .col-sm-3
+          .col-sm-9
+            - if @errors.any?
+              .form__error= t :"start_group_form.form_error"
+
+        - unless user_signed_in?
+          .form-group
+            .col-sm-3
+              %label.control-label{for: 'name'}= t :"start_group_form.name"
+            .col-sm-9
+              - if @errors.include? 'name'
```

Comment:
It's easier than trying to get rails to do this, because name and email are not part of group model.